engine/src/gfx/nav/navscreen.cpp

```diff
@@ -41,6 +41,7 @@
 #include "universe_util.h"
 #include "in_mouse.h"
 #include "gui/glut_support.h"
+#include "networking/netclient.h"
```

Comment:
I believe this file isn't in master any more; it might still be in 0.6.x

Reply:
I'm removing it now. I was building against 0.6.0, as that is what is playable for me...